tests/test_config.py

```diff
@@ -1813,6 +1813,13 @@ def test_multi_model_search_mode(self):
 
         self._evaluate_config(new_args, yaml_content, subcommand='profile')
 
+        # Detailed report generation should fail
+        new_args = deepcopy(args)
+        new_args.append('--create-recommended-detailed-reports')
```

Comment:
Can we add this arg to an L0 test to protect/confirm creation of the reports?

Reply:
Removed `--skip-...` from `L0_profile`